### PR TITLE
fix: lgtm alerts

### PR DESCRIPTION
**Pull request by robertsLando**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
-  cors = require('cors'),
-  ZWaveClient = reqlib('/lib/ZwaveClient'),
-  MqttClient = reqlib('/lib/MqttClient'),
-  Gateway = reqlib('/lib/Gateway'),
-  store = reqlib('config/store.js'),
-  config = reqlib('config/app.js'),
-  debug = reqlib('/lib/debug')('App'),
-  history = require('connect-history-api-fallback'),
-  utils = reqlib('/lib/utils.js');
+const express = require('express')
```
Comment (chrisns) [inline app.js:1]: switching to `const` is unfortunately going to cause pain when we write tests for this and we need to mock dependencies in to unit test. can we stick to `var` or `let` ?

File: lib/ZwaveClient.js [hunk 2]
```diff
@@ -522,23 +520,22 @@ function sceneEvent (nodeid, sceneCode) {
...
   switch (notif) {
     case 0:
-      msg = 'node' + nodeid + ': message complete'
+    // msg = 'node' + nodeid + ': message complete'
       break
     case 1:
-      msg = 'node' + nodeid + ': timeout'
+    // msg = 'node' + nodeid + ': timeout'
       break
     case 2:
```
Comment (chrisns) [inline lib/ZwaveClient.js:531]: this could probably be tidied up a bit?
like:
```js
case 0:
case 1:
case 2:
  break
```

File: lib/ZwaveClient.js [hunk 3]
```diff
@@ -558,7 +555,7 @@ function notification (nodeid, notif, help) {
 
       break
     default:
```
Comment (chrisns) [inline lib/ZwaveClient.js:557]: `default` seems redundant?

File: lib/ZwaveClient.js (hunk 2 above)
Comment (robertsLando) [reply lib/ZwaveClient.js:531]: sure

File: app.js (hunk 1 above)
Comment (robertsLando) [reply app.js:1]: If it causes problems when writing tests I will use var instead, let doesn't have any sense as them are global

**Pull request merged**